### DISCUSSION TITLE
Image base64 fails to save

### DISCUSSION THREAD

**Discussion by codingmachine16** [GrapesJS/grapesjs#3705] [Q&A]
Hi @artf 

 I am really grateful that you helped alot and we are just inches away launching a big project that uses Grapesjs. 

I am having an issue and I know it is not in the scope of grapes js but you can give me a hint as you did in previsio cases and it solved the issue

The issue is that 
 when we upload an image using grapes js uploader , it uploads in one scenario and does not in anothe
1- when we put url of an ethernal image and try to save in db using node js , it works fine

2- when we upload an image from disk and try to save it using node js , it gives cors error though we are handling cors in node js 

Please help me why it fails when we upload from disk?

Thanks alot

**Comment by artf** [comment]
>  it gives cors error

I'm not sure this is related to the editor itself, paste here the error.

**Reply by codingmachine16** [reply]
Thanks @artf 

Issue resolved . It was payloadtoolargeerror request entity too large error. I added below lines in node js express

const app = express();
var bodyParser = require("body-parser");
app.use(bodyParser.json({ limit: "50mb" }));
app.use(
  bodyParser.urlencoded({
    limit: "50mb",
    extended: true,
    parameterLimit: 50000,
  })
);

**Reply by ronaldohoch** [reply]
Have you enabled the cors on express?
https://expressjs.com/en/resources/middleware/cors.html
